docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -171,44 +197,44 @@ Using the dot notation::
 Transforming variables with Jinja2 filters
 ==========================================
 
-Jinja2 filters let you transform the value of a variable within a template expression. For example, the ``capitalize`` filter capitalizes any value passed to it; the ``to_yaml`` and ``to_json`` filters change the format of your variable values. Jinja2 includes many `built-in filters <http://jinja.pocoo.org/docs/templates/#builtin-filters>`_ and Ansible supplies many more filters. See :ref:`playbooks_filters` for examples.
+Jinja2 filters let you transform the value of a variable within a template expression. For example, the ``capitalize`` filter capitalizes any value passed to it; the ``to_yaml`` and ``to_json`` filters change the format of your variable values. Jinja2 includes many `built-in filters <http://jinja.pocoo.org/docs/templates/#builtin-filters>`_ and Ansible supplies many more filters. To know of more examples of filters, see :ref:`playbooks_filters`.
 
 .. _setting_variables:
 
-Where to set variables
-======================
+Where do I define the variables?
+================================
 
-You can set variables in a variety of places, including in inventory, in playbooks, in re-usable files, in roles, and at the command line. Ansible loads every possible variable it finds, then chooses the variable to apply based on :ref:`variable precedence rules <ansible_variable_precedence>`.
+You can define variables in a variety of places, such as in an inventory, in playbooks, in reusable files, in roles, and at the command line. Ansible loads every possible variable it finds, then chooses the variable to apply based on :ref:`variable precedence rules <ansible_variable_precedence>`.
 
 .. _variables_in_inventory:
 
-Setting variables in inventory
-------------------------------
+Defining variables in an inventory
```

Comment:
Ansible inventory can be a file, a directory full of files, or a database populated from a script or inventory plugin, so we don't talk about "an inventory," just "inventory."